ext/pdo/pdo.c

```diff
@@ -281,6 +281,40 @@ PDO_API zend_string *php_pdo_int64_to_str(int64_t i64) /* {{{ */
 }
 /* }}} */
 
+/* Convert uint64 to zend_string */
+PDO_API zend_string *php_pdo_uint64_to_str(uint64_t i64) /* {{{ */
+{
+	char buffer[65];
+	char outbuf[65] = "";
+	register char *p;
```

Comment:
```suggestion
	char *p;
```
From my understanding this is pretty meaningless.